src/main/java/frc/robot/subsystems/Pitch.java

```diff
@@ -48,6 +49,13 @@ public void configure() {
 
     pitchConfig.SoftwareLimitSwitch.ForwardSoftLimitEnable = true;
     pitchConfig.SoftwareLimitSwitch.ForwardSoftLimitThreshold = prefPitch.pitchForwardLimit.getValue();
+    // change stator to supply
+    // add new robot constant for current limit
+    pitchCurrentLimitConfig.withSupplyCurrentLimit(constPitch.CURRENT_LIMIT_CEILING_AMPS);
```

Comment:
Please be consistent with constants vs preferences. All of these should be constants or all of them should be preferences

Reply:
nuh uh